src/components/FormValidator.js

```diff
@@ -50,12 +47,13 @@ class FormValidator {
   _setEventListeners() {
     this._inputEls = [...this._form.querySelectorAll(this._inputSelector)];
     this._submitButton = this._form.querySelector(this._submitButtonSelector);
-
     this._inputEls.forEach((inputEl) => {
       inputEl.addEventListener("input", (e) => {
         this._checkInputValidity(inputEl);
         this.toggleButtonState();
       });
+      this._submitButton.classList.add(this._inactiveButtonClass);
+      this._submitButton.disabled = true;
     });
```

Comment:
You have the `reset` method  for that. Please, call it here not to duplicate the same code